src/components/UrlBlacklist/index.jsx

```diff
@@ -0,0 +1,288 @@
+  const [listuser, setListuser] = useState([]);
+  const [addlink, setAddlink] = useState('');
+  const [dateInput, setDateInput] = useState('');
+  const [datesearch, setDatesearch] = useState('');
+  const [deleteUrl, setDeleteUrl] = useState({});
+  const [deleteMode, setDeleteMode] = useState(false);
+  // const dispatch = useDispatch();
+  const [pageCount] = useState([1, 2, 3, 4]);
+
+  useEffect(() => {
+    const list = [
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh1',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh2',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh3',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh4',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh5',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh6',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh7',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh8',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdh9',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhA',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhB',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhC',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhD',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhE',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+      {
+        link: 'https://longlinkaaaaaaaaaaaa.COM/d213gd621y3hgdhF',
+        addedat: '20:30 - 17/06/2023',
+        addedby: 'Tran Quoc Hieu',
+      },
+    ];
+    setListuser(list);
+  }, []);
+
+  const handleAdding = () => {
```

Comment:
như này là oke rồi nha em
```suggestion
  const handleAdd = () => {
```